cmake/findDependencies.cmake

```diff
@@ -10,6 +10,14 @@ endif()
 
 find_package(Threads REQUIRED)
 
+if(MINGW)
+    find_package(PkgConfig REQUIRED)
+    pkg_check_modules(REGX REQUIRED regex)
```

Comment:
[minor] Could you please change `REGX` to `REGEX`? With just one additional character we can improve the readability of the code 😉

Reply:
@piponazo `REGEX` has been recognized as a keyword by Notepad++, so I didn't even tried to use it... Fixed in commit 815a686